simtools/util/names.py

```diff
@@ -103,6 +106,30 @@
     "OBSLEV": "corsika_obs_level",
 }
 
+corsika7_event_header = {
```

Comment:
See my comments above about this dict. Also, I think it's better to use the string version of the units rather than the actual units like they do in corsikaio.

Reply:
We can use the entire header as you want but I would not stick to only that. I want to provide something more specific to the user, e.g., to provide dedicated function to return magnetic field for example. Hence, I wrote down the parameters that I think are interesting for us. I do not attempt to cover all the possible parameters though, which is rather the approach used in corsikaio. Moreover, how we internally call it in gammasim-tools is independent from other sources. The problem would be changing the naming in the header of CORSIKA, for example, after a CORSIKA update. In that case we need to change here too (but also in corsikaio).

Reply:
I now provide functions to read any of the parameters from the header, read according to the definition from corsikaio, but I still preferred to keep some dedicated functions for B field, interaction height etc. Please have a look, thanks!